### DISCUSSION TITLE
No EPS or Alerts

### DISCUSSION THREAD

**Discussion by yomaoni** [Security-Onion-Solutions/securityonion#7125] [Unsupported Versions] [answered]
Greetings,

I have just done a clean install of SO 2.3.100. I have a management server and 7 sensors. Currently I only have 2 of them monitoring traffic.  In Grafana, I am seeing monitor traffic of 6Gb/s on one and about 1Gb/s on the other.  When I look in the Grid, I am seeing EPS of 0 and no alerts are being generated.  I confirmed that I am able to ping from the manager and the initial setup with off without a hitch but I am just not getting any logs or alerts of any kind.  Any suggestions on what I can do for troubleshooting?

Thanks for your help

**Comment by dougburks** [comment, marked as answer]
If you don't have a search node, then your logs are likely sitting in the redis queue on the manager. 

From https://docs.securityonion.net/en/2.3/architecture.html#distributed:
![image](https://user-images.githubusercontent.com/1659467/152787766-ef35746b-3c14-4d3b-8c63-6b9de982a2a2.png)

**Reply by yomaoni** [reply]
Thanks doug

was afraid of that. Did the wrong install.
